packages/datadog_flutter_plugin/lib/src/tracing/tracing_headers.dart

```diff
@@ -46,66 +47,140 @@ class W3CTracingHeaders {
   static const tracestate = 'tracestate';
 }
 
-enum TraceIdRepresentation {
+/// Controls how we print a TracingId
+enum TracingIdRepresentation {
+  /// Decimal string representation of the tracing id
   decimal,
+
+  /// The low 64-bits of rhe tracing id as a decimal
+  lowDecimal,
+
+  /// Hexadecimal string representation of the full tracing id
   hex,
+
+  /// Hexadecimal string representation of the low 64-bits of the tracing id
   hex16Chars,
+
+  /// Hexadecimal string representation of the high 64-bits of the tracing id
+  highHex16Chars,
+
+  /// Hexadecimal string representation of the full 128-bits of the tracing id
   hex32Chars,
 }
 
+/// Type alias for [TracingIdRepresentation] to ensure backwards
+/// compatibility with other packages
+@Deprecated('Use [TracingIdRepresentation] instead.')
+typedef TraceIdRepresentation = TracingIdRepresentation;
+
+// A value to mask the high 64-bits from a 128-bit trace id.
+@visibleForTesting
+final lowTraceMask = (BigInt.from(0xffffffff) << 32) + BigInt.from(0xffffffff);
+
+/// [TracingId] is used as both a unsigned 64-bit "Span Id" and unsigned 128-bit "Trace Id"
 @immutable
-class TracingUUID {
-  // Because TraceIDs are unsigned and Dart ints are signed, we store the trace id as a BigInt.
-  // Also this will make it easier to switch to 128-bit trace ids at a later date.
+class TracingId {
+  // Because Span Ids are unsigned and Dart ints are signed, we have to store the id as a BigInt
+  // to be able to store all 64-bits properly.
   final BigInt value;
 
-  const TracingUUID(this.value);
+  const TracingId(this.value);
 
-  static TracingUUID fromString(
-      String? uuid, TraceIdRepresentation representation) {
+  static TracingId fromString(
+      String? uuid, TracingIdRepresentation representation) {
```

Comment:
/nit

Not a UUID - just an id.

Reply:
👍 Changed the type but not the parameter. Will fix.